tests/cli/test_repos_sanity.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+CLI="${CLI:-./src/bin/composer-cli}"
+
+rlJournalStart
+    rlPhaseStartSetup
+        repodir_backup=$(mktemp -d composerrepos-XXXXX)
+
+        if [[ -z "$CLI" || "$CLI" == "./src/bin/composer-cli" ]]; then
+            composer_params=$(ps ax | grep -v grep  | grep -m 1 -Po 'bin/lorax-composer \K.*')
+        else
+            composer_params="/var/lib/lorax/composer/blueprints"
+        fi
+        composer_stop
+
+        rlRun -t -c "mv /var/lib/lorax/composer/repos.d/* $repodir_backup"
+        rlRun -t -c "mv /etc/yum.repos.d /etc/yum.repos.d.disabled"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Run lorax-composer with --no-system-repos option"
+        composer_start --no-system-repos $composer_params &
+        wait_for_composer
+        # check that there are no composer repos available
+        rlRun -t -c "$CLI sources list | grep -v '^$' | wc -l | grep '^0$'"
+        rlRun -t -c "$CLI compose start example-http-server qcow2 2>&1 | grep -q \
+            'Problem depsolving example-http-server:'"
```

Comment:
If you don't rename `/etc/yum.repos.d` then you can iterate over all of the .repo files and assert they are not present here. This will make sure the depsolve error isn't due to something else.